security.md

```diff
@@ -97,7 +97,7 @@ Plugins and themes can modify existing, as well as add additional types of, user
 
 ## HTTPS and TLS / SSL
 
-WordPress is fully [compatible with HTTPS when an TLS / SSL certificate](https://make.wordpress.org/support/user-manual/web-publishing/https-for-wordpress/) is installed and available for the web server to use. Support for HTTPS is strongly recommended to help maintain the security of both WordPress logins and site visitors.
+WordPress is [fully compatible with HTTPS](https://make.wordpress.org/support/user-manual/web-publishing/https-for-wordpress/) when an SSL certificate is installed and available for the web server to use. Support for HTTPS is strongly recommended to help maintain the security of both WordPress logins and site visitors.
```

Comment:
This looks like it has an unrelated change in it.

Reply:
the missing TLS?

Reply:
fixed